.codecov.yml

```diff
@@ -0,0 +1,9 @@
+comment: false
+coverage:

```

Comment:
i think it's my fault, edited some files with vim and vim  on windows uses dos line endings b default